flow/flow.go

```diff
@@ -387,22 +386,23 @@ func GetFirstLayerType(encapType string) (gopacket.LayerType, layers.LinkType) {
 
 // LayersPath returns path and the application of all the layers separated by a slash.
 func LayersPath(ls []gopacket.Layer) (string, string) {
```

Comment:
consider doing the same, i.e., using strings builder in the ebpf counterpart. I think this is a significant change since strings do affect performance a lot so why not do it for ebpf as well.

Reply:
you're will do it while rebasing